newrelic/resource_newrelic_entity_tags.go

```diff
@@ -303,3 +305,14 @@ func getTag(tags []*entities.TaggingTagInput, key string) *entities.TaggingTagIn
 
 	return nil
 }
+
+func CreateErrorDiagnostics(res *entities.TaggingMutationResult) diag.Diagnostics {
```

Comment:
this function can be named something like

```suggestion
func handleEntityTagsMutationEmbeddedErrors(res *entities.TaggingMutationResult) diag.Diagnostics {
```